codeHF/dpl-config_run5_hf.json

```diff
@@ -6,6 +6,24 @@
         "step-value-enumeration": "1",
         "aod-file": "@list_o2.txt"
     },
+    "o2-analysis-timestamp": {
+        "isRun2MC": "false"
```

Comment:
Is that a correct device name? From the task, I would guess it should be `timestamp-task`.

Reply:
Fixed in new commit.